### PR TITLE
chore(login): rewrite cockpit login page

### PR DESCRIPTION
**Pull request by marstamm**
related to CAM-12616

### CODE REVIEW

File: ui/cockpit/package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "jquery-ui": "1.12.1",
     "moment": "2.24.0",
     "react": "16.12.0",
+    "react-bootstrap": "1.3.0",
```
Comment (marstamm) [inline ui/cockpit/package.json:35]: Using a UI lib is still up to debate

File: ui/cockpit/src/components/Login/Login.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        <CamundaLogo />
+        <div className="app-name">Cockpit</div>
+      </div>
+
+      <form onSubmit={handleSubmit}>
+        <FormControl
+          placeholder={translate("PAGE_LOGIN_USERNAME")}
+          type="text"
+          value={username}
+          onChange={e => setUsername(e.target.value)}
```
Comment (tasso94) [inline ui/cockpit/src/components/Login/Login.js:85]:
```suggestion
          onChange={e => setUsername(e.target.value)}
          required
```

File: ui/cockpit/src/components/Login/Login.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        ></FormControl>
+        <FormControl
+          placeholder={translate("PAGE_LOGIN_PASSWORD")}
+          type="password"
+          value={password}
+          onChange={e => setPassword(e.target.value)}
```
Comment (tasso94) [inline ui/cockpit/src/components/Login/Login.js:91]:
```suggestion
          onChange={e => setPassword(e.target.value)}
          required
```

**Review by tasso94 — CHANGES_REQUESTED** [review]
Looks good to me. Good job! 

I found two regressions:
* Form does not clear itself after submitting wrong credentials
* Notification "Login failed" previously showed "Process engine with name engine3 does not exist"; this notification does not exist anymore

**Review by tasso94 — APPROVED** [review]
LGTM! 🚀

**Pull request merged**